### PR TITLE
Fingerprint brazilian VW TINGUAN_MK2

### PR DESCRIPTION
**Pull request by AlexandreSato**

### CODE REVIEW

**Comment by pd0wm** [comment]
What is your dongle id?

**Comment by jyoung8607** [comment]
If this is the same one from MadMaxBR on Discord, he says it's a 2019. For that we'll also want a README update expanding the Tiguan's supported model-year range to 2019-20. I don't have a confirmed dongle ID from him and haven't reviewed.

**Comment by estratecnoloiga** [comment]
> What is your dongle id?

7b59e6ded9845ac1

**Comment by AlexandreSato** [comment]
> What is your dongle id?

7b59e6ded9845ac1

File: selfdrive/car/volkswagen/values.py [hunk 1]
```diff
@@ -344,10 +344,12 @@ class CAR:
     (Ecu.eps, 0x712, None): [
       b'\xf1\x875Q0909143M \xf1\x892041\xf1\x820529A6060603',
       b'\xf1\x875QF909144B \xf1\x895582\xf1\x82\00571A60634A1',
+      b'\xf1\x875QM909144B \xf1\x891081\xf1\x82\00521A60604A1',
```
Comment (adeebshihadeh) [inline selfdrive/car/volkswagen/values.py:347]:
```suggestion
      b'\xf1\x875QM909144B \xf1\x891081\xf1\x82\x0521A60604A1',
```

**Pull request merged**